lib/main.coffee

```diff
@@ -11,6 +11,8 @@ Config = require './config'
 KernelManager = require './kernel-manager'
 Inspector = require './inspector'
 AutocompleteProvider = require './autocomplete-provider'
+zmq_services = require './zmq-services'
```

Comment:
One package in nteract is using `jupyter-js-services` as a prototype for using nteract UIs on top of Jupyter notebook backends. I can't comment much on `jupyter-js-services` as I haven't been using it. @jdfreder may know better.